lib/vm/Cargo.toml

```diff
@@ -21,6 +21,7 @@ more-asserts = "0.2"
 cfg-if = "0.1"
 backtrace = "0.3"
 serde = { version = "1.0", features = ["derive", "rc"] }
+rkyv = "0.4.1"
```

Comment:
We should make this an optional dependency

Reply:
Sounds good, will make serde as default, right now it replaced serde

Reply:
Was also consider make it optional in engine-native and user can select to compile, but in that case it has to expose a "use-bincode"/"use-rkyv" feature in lib/api, which seems make wasmer configuration over complicated there, so i only make this optional in lib/vm